src/instance.h

```diff
@@ -45,7 +45,9 @@ class InstanceUpdater : public PollingMetadataUpdater {
  public:
   InstanceUpdater(MetadataAgent* server)
       : reader_(server->config()), PollingMetadataUpdater(
-          server, server->config().InstanceUpdaterIntervalSeconds(),
+          server, 
+          "InstanceUpdater",
```

Comment:
[Optional] You can put this on the previous line.

Reply:
Done.